kymatio/scattering1d/backend/numpy_backend.py

```diff
@@ -111,25 +80,22 @@ def unpad(x, i0, i1):
     return x[..., i0:i1]
 
 
-
-
 def concatenate(arrays):
     return np.stack(arrays, axis=-2)
 
 
 backend = namedtuple('backend',
-                     ['name', 'modulus_complex', 'subsample_fourier', 'real',
+                     ['name', 'modulus', 'subsample_fourier', 
                       'unpad', 'fft', 'concatenate', 'cdgmm'])
 backend.name = 'numpy'
-backend.modulus_complex = modulus
+backend.modulus = modulus
 backend.subsample_fourier = subsample_fourier
-backend.real = real
 backend.unpad = unpad
 backend.pad = pad
-backend.pad_1d = pad_1d
 backend.cdgmm = cdgmm
 backend.fft = FFT(lambda x:scipy.fftpack.fft(x),
+                  lambda x:scipy.fftpack.fft(x),
                   lambda x:scipy.fftpack.ifft(x),
                   lambda x:np.real(scipy.fftpack.ifft(x)),
```

Comment:
are we certain there is no `rifft` or something primitive in `scipy`?

Reply:
There is https://docs.scipy.org/doc/scipy-0.18.1/reference/generated/scipy.fftpack.irfft.html#scipy.fftpack.irfft, however the last time I tried it out I couldn't get the output shapes/outputs to match. I'll try again.